### ISSUE TITLE
Does this run on Tizen (ARM)?

### ISSUE THREAD

**Issue by TheisStellaCare** [Azure/azure-sdk-for-c#1458]
**Query/Question**
Hi I want to compile this for Tizen (For reference, Tizen is the OS that runs on Smart Watches like the Samsung galaxy watch active 2) 

in my case I want to implement this in my own code.
I's there any help for that?

***Why is this not a Bug or a feature Request?***
Cause I'm asking for help

**Setup (please complete the following information if applicable):**
- OS: [e.g. iOS] My dev OS: Windows 10 20H2  -  Target OS: Tizen 4.0
- IDE: [e.g. IntelliJ] Tizen Studio (Modified Eclipse)
- Version of the Library used: Newest

 **Information Checklist**
- [x] Query Added
- [X] Setup information Added

**Comment by danewalton** [comment]
Hey @TheisStellaCare which features are specifically looking to use?

**Comment by TheisStellaCare** [comment]
Hi. Sorry if it wasn't explained very well. 

I'm making a GPS tracker so I need to it send and receive telemetry plus device provisioning

**Comment by danewalton** [comment]
Generally speaking, if you have a way to compile the code it should work just fine. The IoT SDK doesn't do any networking which makes integration on a wider range of platforms easier. 

TLDR: If you have an MQTT stack on your device and can compile the code, it should work just fine!

**Comment by danewalton** [comment]
I'm going to go ahead and close this for now. @TheisStellaCare please let me know if you would like it reopened.
